gradle/libs.versions.toml

```diff
@@ -7,12 +7,21 @@ jline = "3.22.0"
 network = { group = "com.nukkitx.network", name = "raknet", version = "1.6.28-SNAPSHOT" }
 natives = { group = "com.nukkitx", name = "natives", version = "1.0.3" }
 fastutil = { group = "com.nukkitx", name = "fastutil-lite", version = "8.1.1" }
+fastutil1 = { group = "com.nukkitx.fastutil", name = "fastutil-int-short-maps", version = "8.5.3" }
+fastutil2 = { group = "com.nukkitx.fastutil", name = "fastutil-object-int-maps", version = "8.5.3" }
+fastutil3 = { group = "com.nukkitx.fastutil", name = "fastutil-object-object-maps", version = "8.5.3" }
 guava = { group = "com.google.guava", name = "guava", version = "30.1.1-jre" }
 gson = { group = "com.google.code.gson", name = "gson", version = "2.10.1" }
 snakeyaml = { group = "org.yaml", name = "snakeyaml", version = "1.33" }
 leveldb = { group = "org.iq80.leveldb", name = "leveldb", version = "0.11-SNAPSHOT" }
+leveldbjni = { group = "net.daporkchop", name = "leveldb-mcpe-jni", version = "0.0.10-SNAPSHOT" }
+snappy = { group = "org.xerial.snappy", name = "snappy-java", version = "1.1.10.5" }
+expiringmap = { group = "net.jodah", name = "expiringmap", version = "0.5.11" }
```

Comment:
Gson already has an expiring map. Unless necessary we shouldn't be adding more dependencies.